docs/fr/README.md

```diff
@@ -0,0 +1,354 @@
+## Usage
+
+#### Usage De Base
+
+L'utilisation du Générateur de QrCode est très simple. La syntaxe minimale est :
+
+	QrCode::generate('Transformez-moi en QrCode !');
+
+Cela créera un QrCode qui dit "Transformez-moi en QrCode !"
+
+#### Generate
+
+`Generate` sert à fabriquer un QrCode.
+
+	QrCode::generate('Transformez-moi en QrCode !');
+
+>Attention! Cette méthode doit être appelée en dernier si vous l'utilisez dans un appel chaîné.
+
+Par défaut, `Generate` retournera le contenu d'une image SVG sous forme de chaîne. Vous pouvez afficher cette image directement avec un navigateur moderne dans un template Blade de Laravel de cette façon :
+
+	{!! QrCode::generate('Transformez-moi en QrCode !'); !!}
+
+La méthode `generate` accepte un second paramètre pour définir un nom de fichier où enregistrer le QrCode.
+
+	QrCode::generate('Transformez-moi en QrCode !', '../public/qrcodes/qrcode.svg');
+
+#### Changement De Format
+
+>Le générateur de QrCode est prévu pour retourner une image SVG par défaut.
+
+>Attention! La méthode `format` doit être appelée avant toute autre option de formatage, telles que `size`, `color`, `backgroundColor` et `margin`.
+
+Trois formats sont actuellement supportés : PNG, EPS et SVG. Pour changer de format, utilisez le code suivant : 
+
+	QrCode::format('png');  // retourne une image PNG
+	QrCode::format('eps');  // retourne une image EPS
+	QrCode::format('svg');  // retourne une image SVG
+
+#### Changement De Taille
+
+>Le générateur de QrCode retournera par défaut le QrCode dans la plus petite taille possible en pixels.
+
+Vous pouvez changer la taille du QrCode par la méthode `size` qui prend comme paramètre la taille désirée en pixels :
+
+	QrCode::size(100);
+
+#### Changement De Couleur
+
+>Changez les couleurs de vos QrCode avec précaution car certains lecteurs rencontrent des difficultés avec les QrCodes en couleur.
+
+Toutes les couleurs doivent être exprimées en RGB (rouge, vert, bleu). Vous pouvez changer la couleur de trait du QrCode par la méthode `color` :
+
+	QrCode::color(255,0,255);
+
+La couleur de fond peut être définie de la même façon par la méthode `backgroundColor` :
+
+	QrCode::backgroundColor(255,255,0);
+
+#### Changement Des Marges
+
+Vous pouvez définir une marge autour du QrCode par la méthode `margin` :
+
+	QrCode::margin(100);
+
+#### Correction D'Erreur
+
+Il est très aisé de changer le niveau de correction d'erreur. Utilisez la syntaxe suivante :
+
+	QrCode::errorCorrection('H');
+
+Voici la liste des options supportées pour la méthode `errorCorrection`.
+
+| Correction d'Erreur | Capacité De Correction |
+| --- | --- |
+| L | 7% de redondance. |
+| M | 15% de redondance. |
+| Q | 25% de redondance. |
+| H | 30% de redondance. |
+
+>L'élévation du niveau de correction d'erreur se fait au détriment de la taille du QrCode et de la quantité de données qu'il peut stocker. Pour en savoir plus, consultez [error correction](http://en.wikipedia.org/wiki/QR_code#Error_correction) (en anglais).
```

Comment:
You could have easily linked to the french page here (https://fr.wikipedia.org/wiki/Code_QR#Correction_d.27erreur)